src/operator/nn/mkldnn/mkldnn_pooling.cc

```diff
@@ -30,6 +30,11 @@
 namespace mxnet {
 namespace op {
 
+// This can be replaced by mkldnn::memory::desc::data_type() after 1.2 release
```

Comment:
The master branch is using 1.2.2 release of DNNL now, so let's use `data_type()` instead.